modules/renter/siafile/metadata.go

```diff
@@ -0,0 +1,288 @@
+package siafile
+
+import (
+	"math"
+	"os"
+	"time"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/crypto"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+type (
+	// Metadata is the metadata of a SiaFile and is JSON encoded.
+	Metadata struct {
+		version      [16]byte          // version of the sia file format used
+		fileSize     int64             // total size of the file
+		masterKey    crypto.TwofishKey // masterkey used to encrypt pieces
+		pieceSize    uint64            // size of a single piece of the file
+		trackingPath string            // file to the local copy of the file used for repairing
+		siaPath      string
```

Comment:
I think it's worth having a comment here too, someone completely new to the codebase is not going to understand immediately the difference between the siapath and the localpath.